commands/channel_e2e_test.go

```diff
@@ -0,0 +1,86 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package commands
+
+import (
+	"github.com/spf13/cobra"
+
+	"github.com/mattermost/mattermost-server/v5/model"
+
+	"github.com/mattermost/mmctl/client"
+	"github.com/mattermost/mmctl/printer"
+)
+
+func (s *MmctlE2ETestSuite) TestCreateChannelCmd() {
+	s.SetupTestHelper().InitBasic()
+
+	s.RunForAllClients("create channel successfully", func(c client.Client) {
+		printer.Clean()
+
+		cmd := &cobra.Command{}
+		channelName := model.NewRandomString(10)
+		teamName := s.th.BasicTeam.Name
+		channelDisplayName := "channelDisplayName"
+		cmd.Flags().String("name", channelName, "channel name")
+		cmd.Flags().String("team", teamName, "team name")
+		cmd.Flags().String("display_name", channelDisplayName, "display name")
+
+		err := createChannelCmdF(c, cmd, []string{})
+		s.Require().Nil(err)
+		s.Require().Len(printer.GetErrorLines(), 0)
+		s.Require().Len(printer.GetLines(), 1)
+
+		printerChannel := printer.GetLines()[0].(*model.Channel)
```

Comment:
@mgdelacroix - quick question. I see that some tests cast a line to a model object, and some tests actually verify the sentence displayed. In this case, it is "New channel {{.Name}} successfully created". Which is the recommended approach here?